### ISSUE TITLE
(Pre)release on PyPi

### ISSUE THREAD

**Issue by adi-** [django-backup/django-backup#11]
Current dev version is quite stable, so it would be nice to have a pre/final release on PyPi. I just hate adding editable repo links to requirements. It just makes a mess! Can't be easly upgraded etc. Please, think about that.

**Comment by adi-** [comment]
Andy, could you release version 2 on PyPI, please?

**Comment by andybak** [comment]
We've actually just spotted a couple of serious bugs that have been recently introduced. As soon as we've fixed those, I'll do a release.

**Comment by zerok** [comment]
@andybak which are these? Are there entries from them available here in the bug tracker that I've simply missed? 🙂

**Comment by andybak** [comment]
No. We're not 100% sure if they are config specific yet so they are just in our internal tracker until we've investigated. We're all still on holiday at the moment. :)

**Comment by andybak** [comment]
Apologies for the delay. It's been a busy start to the year.

**Comment by andybak** [comment]
Again - apologies for this. If anyone wants to be a co-maintainer then I'm happy to add them on PyPi. All that's needed for a release is a fix for the failing tox tests.

**Comment by zerok** [comment]
No worries :) I will try to look into the failing test this weekend. Completely missed them. Sorry about that 😞
